python/ray/spark/__init__.py

```diff
@@ -425,7 +424,8 @@ def ray_cluster_job_mapper(_):
         #  it might cause Raylet to have a port conflict, likely due to a race condition.
         #  A sleep is added here to attempt to avoid resource allocation contention with available
         #  ports.
-        time.sleep(task_id + 5.0)
+        if sys.platform.startswith("linux"):
```

Comment:
We don't need this if condition, in non-linux system we raises error when importing ray.spark module

Reply:
removed